src/assets/stylesheets/profile.scss

```diff
@@ -101,6 +102,8 @@
       &__icon {
 	@extend %fa-icon-button-icon;
 	background: transparent;
```

Comment:
I was suggesting we drop the border altogether and leave the icon there fully-white -- the borders on these tiny icons are just very visually noisy. (I'm starting to think we should just drop them in the footer menu too, but I think they don't look too bad there.) The downside of removing the borders is just that the border has been the "affordance" (if you can call it that) that it's a button. But it kind of breaks down on these very small buttons.

Reply:
Actually after playing around with it, I think just changing this to have `color: $light-grey` looks pretty good

Reply:
basically if you change `color: $light-grey` these buttons have a new previously unused visual style -- but I think its fine. they are effectively "secondary action buttons" -- I could imagine us re-using that elsewhere. (For example, we may want to restyle the "hamburger menu" button this way)